### ISSUE TITLE
Inconsistent border radios in left/middle rows

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#6200]
### Steps to reproduce

1. Nothing specific to prepare but at least one account and some messages

### Expected behavior

Consistent design

### Actual behavior

![Bildschirmfoto vom 2022-04-07 20-45-57](https://user-images.githubusercontent.com/1374172/162275014-fd92e807-d667-49bb-b66b-36d4748ea7f2.png)
![Bildschirmfoto vom 2022-04-07 20-46-09](https://user-images.githubusercontent.com/1374172/162275012-dafdcba8-389f-46e2-88de-112ef3cbb7a4.png)

### Mail app version

1.x

**Comment by JuliaKirschenheuter** [comment]
@nimishavijay please give a feedback, which design should we use?

**Comment by nimishavijay** [comment]
Good point! We should change the component itself so that it is reflected across all apps :)

My vote would be:
- `border-radius: 6px` for each item in the action menu
- `border-radius: 8px` for the action menu itself
- `padding: 4px` for the action menu 

It would look like this
![image](https://user-images.githubusercontent.com/52440189/165075790-7b777363-e306-4cf5-a470-db3954fa22db.png)
What do you think? cc @jancborchardt @marcoambrosini

**Comment by jancborchardt** [comment]
Yeeeeah that looks sick @nimishavijay! :)

Regarding the AppNavigation, there’s https://github.com/nextcloud/nextcloud-vue/issues/2460 which @quentinguidee / @marcoambrosini wanted to look at, so we should make sure it’s the same. :)

**Comment by ChristophWurst** [comment]
This is too big of a change to be delivered with v1.12

**Comment by jancborchardt** [comment]
This seems to be fixed with the 2 Vue component pull requests, is it not?

**Comment by GretaD** [comment]
all background border corner radius are consistent everywhere in mail now
